compute/cloud-client/src/test/java/compute/SnippetsIT.java

```diff
@@ -105,6 +109,10 @@ public static void setUp()
     compute.CreateInstance.createInstance(PROJECT_ID, ZONE, MACHINE_NAME_WAIT_FOR_OP);
     compute.CreateEncryptedInstance
         .createEncryptedInstance(PROJECT_ID, ZONE, MACHINE_NAME_ENCRYPTED, RAW_KEY);
+    CreateRegionalInstanceTemplate
+        .createRegionalInstanceTemplate(PROJECT_ID, REGION, REGIONAL_LOCATION_NAME);
+    assertThat(stdOut.toString()).contains("Instance Template Operation Status: DONE");
```

Comment:
Kindly refrain from using stdOut to assert. Instead make an API call to assert if the instance is created. Applies to multiple places.

Reply:
Thank you! Creation is tested in testGetRegionalInstanceTemplate, deletion is tested in ReservationIT class.